src/Kotlin/Hello, World!.kt

```diff
@@ -0,0 +1,9 @@
+/**
```

Comment:
Quelltext gehört in `src/main/kotlin`, nicht `src/Kotlin`. Und tu dir selber einen gefallen und nimm Dateinamen ohne Leer- und Sonderzeichen. :wink: